src/libraries/System.IO.Ports/pkg/runtime.native.System.IO.Ports.props

```diff
@@ -17,12 +17,43 @@
     <PackageReadmeFilePath>$(MSBuildThisFileDirectory)../src/PACKAGE.md</PackageReadmeFilePath>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageSupportedRID Include="android-arm" />
+    <PackageSupportedRID Include="android-arm64" />
+    <PackageSupportedRID Include="android-x64" />
+    <PackageSupportedRID Include="android-x86" />
+    <PackageSupportedRID Include="linux-arm" />
+    <PackageSupportedRID Include="linux-arm64" />
+    <PackageSupportedRID Include="linux-bionic-arm64" />
+    <PackageSupportedRID Include="linux-bionic-x64" />
+    <PackageSupportedRID Include="linux-musl-arm" />
+    <PackageSupportedRID Include="linux-musl-arm64" />
+    <PackageSupportedRID Include="linux-musl-x64" />
+    <PackageSupportedRID Include="linux-x64" />
+    <PackageSupportedRID Include="maccatalyst-arm64" />
+    <PackageSupportedRID Include="maccatalyst-x64" />
+    <PackageSupportedRID Include="osx-arm64" />
+    <PackageSupportedRID Include="osx-x64" />
+
+    <!-- We need to add a placeholder file for all RIDs except the one that is actually included in the RID-specific package.
+         This prevents an issue where during publish the SDK would select e.g. both linux-x64 and linux-musl-x64 assets because of the fallback path in the RID graph. -->
+    <PackagePlaceholderRID Include="@(PackageSupportedRID)" Exclude="$(OutputRID)" />
+  </ItemGroup>
+
   <ItemGroup>
     <None Include="$(NativeBinDir)$(LibPrefix)System.IO.Ports.Native$(LibSuffix)"
           PackagePath="runtimes/$(OutputRID)/native"
           Pack="true" />
+    <None Include="$(PlaceholderFile)"
+          PackagePath="@(PackagePlaceholderRID->'runtimes/%(Identity)/native')"
+          Pack="true" />
   </ItemGroup>
 
+  <Target Name="ValidatePackageSupportedRID" BeforeTargets="Build">
+    <Error Text="$(OutputRID) is missing from PackageSupportedRID."
```

Comment:
Is this going to cause build break for community supported targets?

Reply:
It shouldn't since we only reference the packages that match OutputRID here: https://github.com/dotnet/runtime/blob/d3cd5b6510871b97056ed421ae84be4805f547e6/src/libraries/oob-src.proj#L18-L21